app/src/main/java/org/mozilla/webmaker/web/javascript/WebAppInterface.java

```diff
@@ -129,7 +128,8 @@ public String getMemStorage(String key) {
     @JavascriptInterface
     public String getMemStorage(String key, final boolean global) {
         if (!global) key = key.concat(mPrefKey);
-        return MemStorage.sharedStorage().get(key);
+        //return MemStorage.sharedStorage().get(key);
+        return WebmakerAPI.instance.getPayloads(key);
```

Comment:
we bind the API instance as `api` on https://github.com/ryanw-se/webmaker-android/blob/migrate-cache/app/src/main/java/org/mozilla/webmaker/web/javascript/WebAppInterface.java#L56 so we should be able to just do `return api.getPaylods(key)` rather than having to tap into the static `instance` variable on the `WebmakeAPI` class